tendenci/themes/t7-base/templates/payments/stripe/payonline.html

```diff
@@ -65,6 +65,7 @@ <h3>{% trans 'Payment Information' %}</h3>
     <div class="form-group">
         <label for="card-element" class="col-sm-3 control-label">
           Credit or debit card:
+          <br><a href="https://stripe.com" target="_blank" rel="noopener"><img src="/static/themes/t7-base/images/Powered-by-Stripe.png" alt="Powered by Stripe" style="max-width: 100%; height: auto; display: inline-block; width: 150px;"></a>
```

Comment:
Hi @b-davies24,

Can you please change the img src to `{% static 'images/Powered-by-Stripe.png' %}` to not have to hard-code the path `/static/themes/t7-base/`because it might be changed in the future?

Reply:
I've updated the source